src/vos/evt_iter.c

```diff
@@ -109,6 +109,30 @@ evt_iter_finish(daos_handle_t ih)
 	return rc;
 }
 
+static int
+evt_rect_sorted_cmp(const struct evt_rect *rt1, const struct evt_rect *rt2)
```

Comment:
Add a comment and change the name
/** For anchor matching, we want to match the rectangle if it's at the same epoch and
 *.  there is overlap.   This ensures we see extents that may get merged into an extent 
  *  we've already seen.
  */
evt_rect_anchor_cmp

I think using sorted is confusing since evt_rect_cmp is sorted.

Reply:
sure